.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
So u added gitignore, but after already uploading them to github, so they are still here, unfortunately